Mathlib/Data/Set/Semiring.lean

```diff
@@ -25,120 +25,143 @@ open Pointwise
 
 variable {α β : Type _}
 
+-- Porting note: mathlib3 uses `deriving Inhabited, PartialOrder, OrderBot`
 /-- An alias for `set α`, which has a semiring structure given by `∪` as "addition" and pointwise
```

Comment:
```suggestion
/-- An alias for `Set α`, which has a semiring structure given by `∪` as "addition" and pointwise
```

Reply:
This is already done.. maybe you were having a look to an old commit.